less/current/base/variables.less

```diff
@@ -63,6 +63,9 @@
 // In common.css font-size: 13px
 @font-size-base:  13px;
 
+@font-size-page-title: 18px;
```

Comment:
I prefer decoupling of semantic labels (e.g. h4) and visual styles (e.g. title and subtitle looks)